src/main/webapp/views/categoryForm.jsp

```diff
@@ -0,0 +1,97 @@
+                <form:errors path="code" cssClass="help-block" element="div"/>
+            </div>
+        </spring:bind>
+
+        <div class="form-check" id="activeFormGroup">
+            <form:checkbox path="active" cssClass="form-check-input"/>
+            <form:label path="active" cssClass="form-check-label lb-md">Categoria ativa?</form:label>
+            <span class="text-muted">Mostra ou deixa de mostrar a categoria na listagem dos alunos, de formação, etc</span>
+        </div>
+
+        <div class="form-group" id="orderFormGroup">
+            <form:label path="order">Ordem</form:label>
+            <form:input path="order" cssClass="form-control"
+                        placeholder="Por exemplo: categoria de ordem 1 aparece antes da categoria de ordem 2"/>
+            <form:errors path="order" cssClass="help-block" element="div"/>
+        </div>
+
+        <div class="form-group">
+            <form:label path="studyGuide">Guia de estudos</form:label>
+            <form:textarea path="studyGuide" cssClass="form-control" rows="2"
+                           placeholder="Um texto apontando para formações para ajudar pessoas perdidas"/>
+            <form:errors path="studyGuide" cssClass="help-block" element="div"/>
+        </div>
+
+        <div class="form-group">
+            <form:label path="iconPath">Caminho do ícone</form:label>
+            <form:input path="iconPath" cssClass="form-control"
+                        placeholder="Por exemplo: /images/categorias/programacao.svg"/>
+            <form:errors path="iconPath" ccssClass="help-block" element="div"/>
+        </div>
+
+        <spring:bind path="colorCode">
+            <div class="form-group ${status.error ? "has-error has-feedback" : ""}">
+                <form:label path="colorCode">Cor</form:label>
+                <form:input path="colorCode" cssClass="form-control" placeholder="Por exemplo: #fcc14a"/>
+                <span class="glyphicon glyphicon-remove form-control-feedback" aria-hidden="true"></span>
+                <form:errors path="colorCode" cssClass="help-block" element="div"/>
+            </div>
+        </spring:bind>
+
+        <div class="form-group">
+            <form:label path="description">Descrição</form:label>
+            <form:input path="description" cssClass="form-control"
+                        placeholder="Por exemplo: iOS, Android, PhoneGap e mais..."/>
+            <form:errors path="description" cssClass="help-block" element="div"/>
+        </div>
+
+        <button type="submit" class="btn btn-primary mb-2" id="submitButton">Enviar</button>
+    </form:form>
+</div>
+
+<footer>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/js/bootstrap.min.js"
+            integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
+            crossorigin="anonymous"></script>
```

Comment:
Acho que você poderia baixar os dois e manter no projeto, assim evita de quebrar seu layout se os CDN externos cairem

Reply:
Não é muito improvável que CDNs vão cair completamente?

Reply:
Sim, mas pensando num projeto real não é bom confiar, uma coisa é se for video ou imagens e tal, mas algo como CSS e javascript, que quebra completamente a exibição da página não é interessante sabe

Reply:
> Sim, mas pensando num projeto real não é bom confiar, uma coisa é se for video ou imagens e tal, mas algo como CSS e javascript, que quebra completamente a exibição da página não é interessante sabe

Faz sentido.